src/universal/modules/teamDashboard/components/ProjectColumn/ProjectColumn.js

```diff
@@ -167,6 +167,9 @@ class ProjectColumn extends Component {
         </div>
         <div className={css(styles.columnBody)}>
           <div className={css(styles.columnInner)}>
+            {(projects.length === 0 && label === 'done') &&
```

Comment:
Ok, I like this iteration of design best. So, it:

It:
- [ ] shows the text only after on-boarding
- [ ] once just after the meeting has ended

I'll update the issue and the point value.